backend/backend/MemoryGameWithNumbers/Controllers/MemoryGameWithNumbersController.cs

```diff
@@ -23,12 +23,24 @@ public MemoryGameWithNumbersController(MemoryGameService memoryGameService)
     [HttpPost("attempt")]
     public ActionResult<bool> CheckAttempt([FromBody] List<int?> userAttempt)
     {
+        if (userAttempt == null || !userAttempt.Any())
```

Comment:
Rider suggestionai xd

- `userAttempt == null` isvis nereikia, nes visada bus false, nes tik list'o elementai nullable, ne pats list, tai expression visada bus false
- `userAttempt.Count == 0` geriau for readability ir performance

Reply:
+++